src/macaron/malware_analyzer/pypi_heuristics/sourcecode/suspicious_setup.py

```diff
@@ -123,3 +124,88 @@ def analyze(self, api_client: PyPIRegistry) -> tuple[HeuristicResult, dict]:
         if suspicious_setup:
             return HeuristicResult.FAIL, {"import_module": imports}
         return HeuristicResult.PASS, {"import_module": imports}
+
+    def extract_from_ast(self, source_content: str) -> set[str]:
+        """Extract imports from source code using the parsed AST.
```

Comment:
We should also document the type of import are ignored here too (i.e the relative imports).